internal/servers/controller/handlers/host_sets/host_set_service.go

```diff
@@ -817,7 +820,10 @@ func toProto(ctx context.Context, in host.Set, hosts []host.Host, opt ...handler
 	switch h := in.(type) {
 	case *plugin.HostSet:
 		if outputFields.Has(globals.PreferredEndpointsField) {
-			out.PreferredEndpoints = h.GetPreferredEndpoints()
+			out.PreferredEndpoints = h.PreferredEndpoints
+		}
+		if outputFields.Has(globals.SyncIntervalSecondsField) {
+			out.SyncIntervalSeconds = &wrapperspb.Int32Value{Value: h.GetSyncIntervalSeconds()}
```

Comment:
consider only setting this if GetSyncIntervalSeconds is not 0.  Since this is in a Int32Value I believe it'll be present despite it being a zero value.

Reply:
Updated